### ISSUE TITLE
React Native version 2.0.0 not working

### ISSUE THREAD

**Issue by alexhackers** [ably/ably-js#1711]
In the new version 2.0.0 the project receive error:

"Unable to resolve "ably" from "App.tsx"

The receive this error in:

const client = new Ably.Realtime({
  key: 'XXXX',
  clientId: 'XXXX',
});

┆Issue is synchronized with this [Jira Task](https://ably.atlassian.net/browse/ECO-4709) by [Unito](https://www.unito.io)

**Comment by ttypic** [comment]
Hey @alexhackers, thank you for bringing this to our attention! Could you share some details? The error description seems to indicate that you're trying to import ably in your App.tsx, but you haven't exported it.

If your `App.tsx` contains code you shared, you probably need to change it like this:

```jsx
export const ably = new Ably.Realtime({
  key: 'XXXX',
  clientId: 'XXXX',
});
```

**Comment by VeskeR** [comment]
Helo @alexhackers , thank you for reporting this issue!

I was able to reproduce it locally following quick start guide for React Native using Expo Go. Will look into fixing it.

**Comment by VeskeR** [comment]
We identified the problem with not being able to import `ably` v2 in React Native apps. I was able to fix the issue locally. It came down to React Native not currently supporting "exports" field in package.json by default when consuming npm packages ([source](https://reactnative.dev/blog/2023/06/21/package-exports-support)) - which we are using in v2. It is still an experimental feature and not enabled by default in new React Native apps. We will fix this issue so ably-js v2 is working with React Native apps without enabling this flag.

Unfortunately, we discovered another issue with using ably-js v2 React Native bundle, which prevents us from using it in React Native apps. It is tracked in github issue https://github.com/ably/ably-js/issues/1712.

Fixing this issue will be our high priority on Monday and we will report back as soon as v2 can be used in React Native.
In the meantime I would suggest installing the latest ably-js v1 package for the React Native app (`npm install ably@1`)

**Comment by Dajust** [comment]
Any updates yet?

**Comment by VeskeR** [comment]
Hello @Dajust . We identified both issues and resolved them locally, working on merging and releasing a patch version now.

**Comment by VeskeR** [comment]
@alexhackers , @Dajust This issue has been fixed in ably v2.0.1. Please try this version and let us know if you have any further issues. Thank you!

**Comment by owenpearson** [comment]
Resolved by #1713
